Userland/Libraries/LibWeb/HTML/Scripting/ImportMap.cpp

```diff
@@ -209,4 +225,47 @@ WebIDL::ExceptionOr<HashMap<URL::URL, ModuleSpecifierMap>> sort_and_normalise_sc
     return normalised;
 }
 
+// https://html.spec.whatwg.org/multipage/webappapis.html#normalizing-a-module-integrity-map
+WebIDL::ExceptionOr<ModuleIntegrityMap> normalise_module_integrity_map(JS::Realm& realm, JS::Object& original_map, URL::URL base_url)
```

Comment:
The same spelling as is used in the specification should be used for "normalize".